src/app/modules/rup/components/elementos/adjuntarDocumento.component.ts

```diff
@@ -3,13 +3,17 @@ import { RUPComponent } from './../core/rup.component';
 import { environment } from '../../../../../environments/environment';
 import { RupElement } from '../elementos';
 import { ISnomedConcept } from '../../interfaces/snomed-concept.interface';
+import { isUndefined } from 'util';
 @Component({
     selector: 'rup-adjuntar-documento',
     templateUrl: 'adjuntarDocumento.html',
     styleUrls: ['adjuntarDocumento.scss'],
 })
 @RupElement('AdjuntarDocumentoComponent')
 export class AdjuntarDocumentoComponent extends RUPComponent implements OnInit {
+    @Input() permiteCarga: boolean;
+    @Input() parametroRegistro;
```

Comment:
Ya sabemos que es un parametro XD
```suggestion
    @Input() registro;
```